protocol/x/clob/keeper/stateful_order_state.go

```diff
@@ -478,3 +489,51 @@ func (k Keeper) getUntriggeredConditionalOrdersIterator(ctx sdk.Context) sdk.Ite
 	)
 	return sdk.KVStorePrefixIterator(store, []byte{})
 }
+
+// GetStatefulOrderCount gets a count of how many stateful orders are written to state. This does not
+// include any untriggered conditional orders.
+//
+// OrderId can be conditional or long term.
+func (k Keeper) GetStatefulOrderCount(
+	ctx sdk.Context,
+	orderId types.OrderId,
```

Comment:
Any reason why we don't pass in a subaccount ID here instead of an order ID?

Reply:
Swapped to subaccount. 

It was mainly to ensure that the counts are only fetched when processing a stateful order but that seems like overkill.